go.mod

```diff
@@ -1,271 +1,322 @@
+	k8s.io/kms v0.32.3 // indirect
+	k8s.io/kube-openapi v0.0.0-20241105132330-32ad38e42d3f // indirect
+	k8s.io/kubelet v0.32.3 // indirect
 	k8s.io/mount-utils v0.0.0 // indirect
 	k8s.io/pod-security-admission v0.0.0 // indirect
-	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2 // indirect
+	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738 // indirect
 	lukechampine.com/uint128 v1.2.0 // indirect
 	modernc.org/cc/v3 v3.40.0 // indirect
 	modernc.org/ccgo/v3 v3.16.13 // indirect
-	modernc.org/libc v1.24.1 // indirect
-	modernc.org/mathutil v1.5.0 // indirect
-	modernc.org/memory v1.6.0 // indirect
+	modernc.org/libc v1.29.0 // indirect
+	modernc.org/mathutil v1.6.0 // indirect
+	modernc.org/memory v1.7.2 // indirect
 	modernc.org/opt v0.1.3 // indirect
-	modernc.org/sqlite v1.25.0 // indirect
+	modernc.org/sqlite v1.27.0 // indirect
 	modernc.org/strutil v1.1.3 // indirect
 	modernc.org/token v1.0.1 // indirect
-	resenje.org/singleflight v0.4.1 // indirect
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.1.2 // indirect
-	sigs.k8s.io/controller-runtime v0.16.2 // indirect
-	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
-	sigs.k8s.io/yaml v1.3.0 // indirect
+	resenje.org/singleflight v0.4.3 // indirect
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.31.0 // indirect
+	sigs.k8s.io/controller-runtime v0.20.4 // indirect
+	sigs.k8s.io/json v0.0.0-20241010143419-9aa6b5e7a4b3 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.4.2 // indirect
+	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
 replace (
 	// kube 1.28.7 is pinned to cel v0.16.1
-	github.com/google/cel-go => github.com/google/cel-go v0.16.1
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.28.7
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.28.7
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.28.7
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.28.7
-	k8s.io/code-generator => k8s.io/code-generator v0.28.7
-	k8s.io/cri-api => k8s.io/cri-api v0.28.7
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.28.7
-	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.28.7
-	k8s.io/endpointslice => k8s.io/endpointslice v0.28.7
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.28.7
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.28.7
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.28.7
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.28.7
-	k8s.io/kubectl => k8s.io/kubectl v0.28.7
-	k8s.io/kubelet => k8s.io/kubelet v0.28.7
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.28.7
-	k8s.io/metrics => k8s.io/metrics v0.28.7
-	k8s.io/mount-utils => k8s.io/mount-utils v0.28.7
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.28.7
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.28.7
+	github.com/google/cel-go => github.com/google/cel-go v0.22.0
```

Comment:
I think this pin can be removed entirely now

Reply:
no, there is a breaking change as well in the latest version :(